hack/bin/integration-cleanup.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Script to delete any helm releases prefixed with `test-` older than 20 minutes deemed inactive
+#
+# Motivation: cleanup of old test clusters that were not deleted (e.g. by the CI system, because it broke)
+
+date_seconds() {
+  d=$1
+  if [[ $OSTYPE =~ darwin* ]]; then
+    date -j -f "%b %d %H:%M:%S %Y" "$date_str" +"%s"
+  else 
+    date -d "$d" +"%s"
+  fi
+}
+
+date_now=$(date +"%s")
+
+# Since helm only shows information as hard-to-parse text pending https://github.com/kubernetes/helm/pull/2950
+# we are stuck with brittle grep/awk magic.
```

Comment:
This issue is resolved btw, maybe we can change this in future.

Reply:
👍🏿 added to FUTUREWORK.